### PR TITLE
Add Google site verification meta

### PR DESCRIPTION
**Pull request by benbalter**
To allow us to register the site with Google webmaster tools to register the sitemap added via https://github.com/jekyll/jekyll/pull/4553.

### CODE REVIEW

File: site/_config.yml [hunk 1]
```diff
@@ -31,3 +31,5 @@ gems:
   - jekyll-sitemap
   - jekyll-seo-tag
   - jekyll-avatar
+
+google_site_verification: onQcXpAvtHBrUI5LlroHNE_FP0b2qvFyPq7VZw36iEY
```
Comment (parkr) [inline site/_config.yml]: Perhaps this should go above with `google_analytics_id`?

**Comment by parkr** [comment]
One thought, otherwise :+1:/

File: site/_config.yml (hunk 1 above)
Comment (benbalter) [inline site/_config.yml]: Good call. b6a0b9e

**Comment by envygeeks** [comment]
:+1:

**Pull request merged**